dom/calculator.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script defer src="exercise6-calculator.js"></script>
+    <link rel="stylesheet" href="calculator.css">
+    <title>Calculator Program</title>
+</head>
+<body>
+    <main>
+        <h1 class="title">Calculator Program</h1>
+
+        <div id="calc-grid-container">
+            <div id="display-box"></div>
+            <input type="button" class="button" id="button0" value="0" onclick="displayValue('0')">
```

Comment:
remove onclick on html elements. Do that in JS please. Also convert to div instead of input

Reply:
converted to div and removed onclick element